moja-kuhinjica-frontend/src/pages/index.tsx

```diff
@@ -93,7 +115,7 @@ const Home = () => {
                                     : styles.dayButton
                             }
                         >
-                            Cetvrtak
+                            Četvrtak
                         </button>
                         <button
                             onClick={() => setActive(5)}
```

Comment:
When you have part of code that is repeating constantly , then you should put it in separate component

Reply:
Done.